tests/PKSim.Tests/Core/DataColumnMapperSpecs.cs

```diff
@@ -8,16 +9,17 @@
 
 namespace PKSim.Core
 {
-   public abstract class concern_for_DataColumnMapper : ContextSpecification<DataColumnMapper>
```

Comment:
@rwmcintosh 
Those tests use a lot of MustHaveBeenCalled etc.. 
Can you refactor to verify that that snapshot property is set propertly instead?
eg.
_snapshot.DataInfo.ShouldBeEqualTo(_snapshotDataInfo)

instead of saying that the mapper map method should have been called. 
In general, tests gets less brittle when we do not rely too much on what calls are being made.